boxoffice/models/discount_policy.py

```diff
@@ -43,6 +45,9 @@ class DiscountPolicy(BaseScopedNameMixin, db.Model):
     percentage = db.Column(db.Integer, nullable=True)
     # price-based discount
     is_price_based = db.Column(db.Boolean, default=False, nullable=False)
+    discount_code_base = db.Column(db.Unicode(20), nullable=True)
```

Comment:
We do fresh installations sometimes to clean up past migration errors that have left production out of sync with code.